### PR TITLE
Emphasizing patient info in tablet mode

### PR DESCRIPTION
**Pull request by jpa399**
_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

Reformatted the patient summary in the patient control panel to make it more emphasized in tablet mode.  I did this by adjusting the grid percentages in the patient control panel and dividing the patient info into a grid as well.  I also moved the modal arrow and all of its functions into the summary header section so that it remained next to the patient name.
## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [ ] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- x ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [x] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

**Comment by sgober** [comment]
[SO1] If you scroll all the way down, the next patient button now detaches from the bottom in both iPad and regular mode.  This does not happen on master.

**Comment by jpa399** [comment]
I think that might be because it needs a rebase.  I'll take care of that right now

**Comment by sgober** [comment]
related to SO1... seems to be happening when the top patient info becomes scrollable.

Screen Shot 2019-07-22 at 1 39 10 PM

File: src/summary/SummaryHeader.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        margin-bottom: 15px;
+    }
+    
+    .tablet-patient-name {
+        font-weight: 600;
+        font-size: 20px !important; 
+    }
+    
+    .tablet-patient-info {
+        color: black !important;
```
Comment (dreeves4321) [inline src/summary/SummaryHeader.scss]: color: $body-gray
size: $size-m

**Comment by dreeves4321** [comment]
[DR1] the items should be color: $body-gray, and font-size: $size-m

**Comment by jpa399** [comment]
[DR1] resolved!

**Comment by jpa399** [comment]
[SO1] resolved!

**Comment by sgober** [comment]
Not sure SO1 is resolved, now the next patient button is cut off at the bottom.  Also see the screenshot below, the dotted line on the right now has no padding with the tabular/narrative toggle buttons.

Screen Shot 2019-07-23 at 8 45 09 AM

**Comment by dreeves4321** [comment]
Why is there a dotted line at the right anyway? There's no split screen mode in the tablet for now.

**Comment by jpa399** [comment]
I think [SO1] should be resolved now.  I removed the gray dashed line as well.

**Comment by sgober** [comment]
I am still seeing the next patient button cut off.  I think it has to do with the fact that the patient control panel is now 107px in height, not the 85px it was before.  If you scroll all the way down after loading, you can see that the top part of the panel is cut off.  This shouldn't be scrollable.

Screen Shot 2019-07-23 at 9 41 59 AM

**Comment by sgober** [comment]
also now needs a rebase

**Comment by jpa399** [comment]
[SO1] resolved?

**Comment by sgober** [comment]
It is! Wahoo!

One nitpick...
[SO2] The right margin has more whitespace than the left now because the dotted line is gone.  Is there any way we can let that out a little bit to match the left?  If it isn't trivial then we don't need to worry about it.

**Comment by sgober** [comment]
[SO3] Small thing... when in portrait mode, the next button is slightly off (probably by like one pixel).  This looks great in landscape mode.  If there isn't a quick fix, I would say move on for now.

Screen Shot 2019-07-24 at 10 34 33 AM
Screen Shot 2019-07-24 at 10 35 53 AM

**Comment by jpa399** [comment]
[SO2] resolved!

**Comment by jpa399** [comment]
I'm not sure how to fix this, and it also seems to be too small to see on my screen so I can't tell if it is fixed or not :/ Do you have any ideas or should I just move on?

**Comment by sgober** [comment]
@jpa399 let me poke at it for 10-15 and see if I can make any headway.  if not, I say we just move on!

**Comment by sgober** [comment]
@jpa399 committed a quick fix that takes care of the problem.  This looks good to me now, just needs a rebase!

**Comment by jafeltra** [comment]
This looks really good! I have one really small thing:
[JA1]: It looks like there is no longer a space between the patient's name and the mrn number on /pilot1.

Besides that, this looks great!

**Comment by jafeltra** [comment]
Oh no I think I lied I found one more thing :( 

[JA2]: It looks like the Patient Control Panel doesn't line up with the top left corner anymore. There seems like there's space there that wasn't there before. I tried to take a screenshot to show the space on the left side, but I can also talk about it tomorrow morning if you want!
image

**Comment by jpa399** [comment]
[JA1] resolved!

**Comment by jpa399** [comment]
[JA2] resolved!

**Comment by jpa399** [comment]
rebased!

**Pull request merged**